core/src/main/java/org/polypheny/db/monitoring/events/QueryDataPoint.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019-2021 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.monitoring.events;
+
+public interface QueryDataPoint extends MonitoringDataPoint {
+
+    long getExecutionTime();
+
+    String getQueryId();
+
+    String getPhysicalQueryId();
```

Comment:
What is (or is there at all) a d difference between physicalQueryId and physicalQueryClass?

Reply:
physicalQueryId was the former name and it should be named **phsyicalQueryClass**. If there are still some queryId in code, this was just overseen during refactoring.

Reply:
Renamed in https://github.com/polypheny/Polypheny-DB/pull/364/commits/9f6e5ffbb6bc17e207517578d513ac89b9f472f6